### PR TITLE
added nft button to youtube

### LINKED ISSUE #5: Web page augmentation for social media sites
We need two features for social media augmentation: 1) Dialog for user connection, 2)Water for video, buttons for NFT Mint and Tips.

- [x] TikTok, 
- [ ] YouTube

### PR DESCRIPTION
**Pull request by c3V6a2Vy**
image

1. added nft button to youtube
2. factored out the nft button to a common place
3. changed nft link from `div` to `button`

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -12,16 +14,36 @@
...
+        "content.styles.css"
+      ]
+    },
+    {
+      "matches": [
+        "https://*.youtube.com/*",
+        "http://*.youtube.com/*"
+      ],
+      "js": [
+        "ytContentScript.bundle.js"
```
Comment (Gassa) [inline src/manifest.json:34]: Not sure if I missed anything, but I don't see any webpack process that emitting this js file

File: src/extension/Content/index.ts [hunk 2]
```diff
@@ -1,28 +1,11 @@
 import { LiveSelector, MutationObserverWatcher } from '@dimensiondev/holoflows-kit'
 import browser from 'webextension-polyfill'
-
-import { TopNavMenuSelector, VideoFeedSelector } from './Selectors/twitter'
-
-// add a link on nav menu
-const navWrapper = new LiveSelector().querySelector<HTMLDivElement>(TopNavMenuSelector).enableSingleMode().evaluate()
-
-if (navWrapper) {
```
Comment (Gassa) [inline src/extension/Content/index.ts:9]: @c3V6a2Vy what is the reason to delete these web injection for tiktok?

Comment (c3V6a2Vy) [reply src/extension/Content/index.ts:9]: Oh I thought the top nav bar were for twitter? Are they not?

File: src/manifest.json (hunk 1 above)
Comment (c3V6a2Vy) [reply src/manifest.json:34]: It’s exported from Line 32 webpack.config

File: webpack.config.js [hunk 3]
```diff
@@ -29,9 +29,10 @@ var options = {
     popup: path.join(__dirname, 'src', 'extension', 'Popup', 'index.jsx'),
     background: path.join(__dirname, 'src', 'extension', 'Background', 'index.ts'),
     contentScript: path.join(__dirname, 'src', 'extension', 'Content', 'index.ts'),
+    ytContentScript: path.join(__dirname, 'src', 'extension', 'Content', 'yt.ts'),
```
Comment (c3V6a2Vy) [inline webpack.config.js:32]: @Gassa exported here

I can change the tt content script to the same naming convention as well

**Pull request closed without merging**